src/lib/common/navigation.ts

```diff
@@ -0,0 +1,14 @@
+import { schema as schemaStore } from "../../stores";
+import { get } from "svelte/store";
+
+export function openBrowsePage() {
+  window.location.href = "browse.html";
+}
+
+export function startSketching(authorityName: string) {
```

Comment:
I think this should take a parameter for the schema. The schema store doesn't make any sense for the browse page; it really should be refactored to something more clearly sketch specific. For this PR I'd just vote to make it a param

Reply:
So do you think this should be an optional param? e.g. if a schema is provided: use that, otherwise, use what's in the store

Reply:
I agree the browse page shouldn't know about schema. Currently the browse page doesn't use this function but I think it should eventually use it for 'edit a local version of the this scheme', or at least use a function from this module. I suppose in that instance you can find out from the scheme you're trying to edit what schema you want.

Reply:
I think it should be a required parameter. So yes, in that second case, it would check if `pipeline` is in the scheme object and decide schema from that